data/json/terrain.json

```diff
@@ -8092,6 +8092,27 @@
       "items": [ { "item": "plastic_chunk", "count": [ 0, 2 ] }, { "item": "scrap", "prob": 50 } ]
     }
   },
+  {
+    "type": "terrain",
+    "id": "t_card_fp",
+    "looks_like": "t_card_science",
+    "name": "card reader",
+    "description": "This looks like a smartcard reader.  You can see \"Fp\" soberly printed on the reader.  The reading surface is pretty big, you have no idea what could open it.",
```

Comment:
Judging from the logic here,
https://github.com/CleverRaven/Cataclysm-DDA/blob/c4bb91a157469597ecd4155352a896901ddb0c31/src/iexamine.cpp#L808-L826
It'll look nothing like a smartcard reader, rather be described to look like a mold for an unusually large, cartoonish face.

Reply:
Or alternatively be described as a device with a large scanning bed at head-level. Maybe mentions a pictogram of a face with an arrow pointing towards the bed.

Reply:
Yep makes sens, it's a left over from the initial copy-paste, I'll fix it.